docs/src/main/asciidoc/kafka.adoc

```diff
@@ -497,6 +497,9 @@ The partitions of the subscribed topic will be divided among the consumer thread
 Note that if the `partitions` value exceed the number of partitions of the topic, some consumer threads won't be assigned any partitions.
 +
```

Comment:
I would add a note that `partitions` is deprecated.

Reply:
I mentioned the `concurrency` and added a deprecation note for `partitions`.